src/lib.rs

```diff
@@ -41,15 +41,21 @@
 //!
 //! # Crate features
 //!
-//! This crate provides one feature named `openai-vocabulary-file`. This feature
-//! bundles the default vocabulary file used for OpenAI's CLIP model together
-//! with this crate and allows users to construct a new tokenizer simply by
-//! calling [`Tokenizer::new`].
+//! This crate provides two features, both of which are enabled by default:
 //!
-//! This feature is enabled by default. To disable it use `default-features =
-//! false` when specifying the dependency on this crate in your `Cargo.toml`.
-//! You will need to supply your own vocabulary file then and construct the
-//! tokenizer using [`Tokenizer::with_vocabulary`].
+//! * **ndarray** - Adds a dependency on the
+//!   [`ndarray`](https://docs.rs/ndarray) crate and enabled the
+//!  [`Tokenizer::tokenize_batch`] method that can be used to tokenize several
+//!  input strings at once and returns a matrix suitable for directly passing to
+//!  the CLIP neural network.
```

Comment:
The "can be used to [..] at once and returns" feels a little awkward. How about, "at once, returning"?